src/core/connectors.js

```diff
@@ -967,11 +967,14 @@ define(function() {
 	},
 	{
 		label: 'Naver',
-		matches: ['*://playerui.music.naver.com*'],
+		matches: ['*://playerui.music.naver.com/*'],
```

Comment:
Looks like I missed that one :/

Reply:
Never Mind, I just looked around and it was a simple error!